lib/syskit/dynamic_port_binding.rb

```diff
@@ -203,6 +203,10 @@ def initialize(
             # Method called by {Accessor} to create the accessor object from a
             # port
             def create_accessor(port)
+                if port.respond_to?(:init_policy) &&
+                   [true, false].include?(port.init_policy)
```

Comment:
Do not check this.

The `init_policy` is a default. Use it as a default unconditionally and let the caller override it via the various `policy` arguments.